client/.storybook/addons.js

```diff
@@ -1,4 +1,6 @@
 /* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
 
-import '@storybook/addon-actions/register';
+// import '@storybook/addon-actions/register';
```

Comment:
Why not remove this completely instead of commenting it out?

Reply:
I'm gonna reinstate this, after reading the docs, I now know what this is for. I'll wire up its usage in all the stories.

Reply:
Cool, added it. It does underscore the problem that some of our components did not properly pass along event handlers, so I've marked them as a TODO and will file issues separately to address them, as well as update the component wiki page to make sure this is something developers think about.